apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/SpreadsheetImport/SpreadsheetImportPreview.tsx

```diff
@@ -103,88 +126,89 @@ const SpreadsheetImportPreview = ({
+                      table due to incompatible headers.
+                    </p>
+                  )}
+                  {!isCompatible && (
+                    <div className="space-y-2">
+                      <div className="flex items-start space-x-2">
+                        <div className="size-[14px] flex items-center justify-center translate-y-[3px]">
+                          <div className="size-[6px] rounded-full bg-foreground-lighter" />
+                        </div>
+                        <p className="text-sm">
+                          The column{incompatibleHeaders.length > 1 ? 's' : ''}{' '}
+                          {incompatibleHeaders.map((x) => `"${x}"`).join(', ')}{' '}
+                          {incompatibleHeaders.length > 1 ? 'are' : 'is'} not present in your table
+                        </p>
                       </div>
-                      <p className="text-sm">
-                        This CSV <span className="text-red-900">cannot</span> be imported into your
-                        table due to incompatible headers:
-                        <br />
-                        The column{incompatibleHeaders.length > 1 ? 's' : ''}{' '}
-                        {incompatibleHeaders.map((x) => `"${x}"`).join(', ')}{' '}
-                        {incompatibleHeaders.length > 1 ? 'are' : 'is'} not present in your table
-                      </p>
                     </div>
-                  </div>
-                )}
-                {errors.map((error: any, idx: number) => {
-                  const key = `import-error-${idx}`
-                  const isExpanded = expandedErrors.includes(key)
+                  )}
+                  <div className="space-y-2">
+                    {dedupedErrors.map((error: any, idx: number) => {
+                      const key = `import-error-${idx}`
+                      const isExpanded = expandedErrors.includes(key)
 
-                  return (
-                    <div key={key} className="space-y-2">
-                      <div
-                        className="flex items-center space-x-2 cursor-pointer"
-                        onClick={() => onSelectExpandError(key)}
-                      >
-                        {error.data !== undefined ? (
-                          <ChevronRight
-                            size={14}
-                            className={`transform ${isExpanded ? 'rotate-90' : ''}`}
-                          />
-                        ) : (
-                          <div className="w-[14px] h-[14px] flex items-center justify-center">
-                            <div className="w-[6px] h-[6px] rounded-full bg-foreground-lighter" />
+                      return (
+                        <div key={key} className="space-y-2">
+                          <div
+                            className="flex items-center space-x-2 cursor-pointer"
```

Comment:
The non-clickable items (no details) also look clickable here, which might be a bit confusing:

![Image](https://github.com/user-attachments/assets/9381abc3-5e69-4637-8651-0414967c1833)